book/run/optimism.md

```diff
@@ -0,0 +1,118 @@
+# Running Reth on OP Stack chains
+
+`reth` ships with the `optimism` feature flag, enabling support for OP Stack chains out of the box. Optimism has a small diff from the [L1 EELS][l1-el-spec], comprising of the following key changes:
```

Comment:
I think it's worthwhile to specify reth **crates** and not the **reth** binary here